skills/declarative/Calendar/Calendar/dialogs/CreateEventDialog/CreateEventDialog.dialog

```diff
@@ -332,15 +335,24 @@
           "$designer": {
             "id": "2yLHBA"
           },
-          "name": "CreateEventDialog.OnEvent_FindMeetingTimes"
+          "name": "CreateEventDialog.OnEvent_FindMeetingTimes",
+          "value": "=null"
```

Comment:
Is this actually needed? I thought it would just default to null anyway

Reply:
No, when its not provided, it sends the full state read out